static/src/javascripts/projects/common/modules/experiments/tests/unruly-performance.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+export const unrulyPerformanceTest: ABTest = {
+    id: 'UnrulyPerformanceTest',
+    start: '2017-11-10',
+    expiry: '2018-02-28',
+    author: 'Francis Carr',
+    description:
+        'This test removes 5% of users from Unruly to measure performance impact',
+    audience: 0.1,
+    audienceOffset: 0,
+    successMeasure: 'No negative impact on attention time',
+    audienceCriteria: 'All web traffic.',
+    dataLinkNames: '',
+    idealOutcome: 'We are informed on the impact of Unruly videos to users',
+    canRun: () => true,
+    variants: [
+        {
+            id: 'serve-unruly',
```

Comment:
did you confirm that these do come through as key values? think it should be fine.

Reply:
@rich-nguyen Yes, they go through to DFP.